core-tests/src/test/java/org/mule/runtime/core/work/MuleEventWorkTestCase.java

```diff
@@ -41,7 +41,7 @@ protected void doSetUp() throws Exception {
 
   @Test
   public void testScheduleMuleEventWork() throws Exception {
-    muleContext.getWorkManager().scheduleWork(new TestMuleEventWork(originalEvent));
+    muleContext.getSchedulerService().computationScheduler().submit(new TestMuleEventWork(originalEvent));
```

Comment:
Any pool is the same here, since just a test task is scheduled. This task just performs some assertions.